staging/src/k8s.io/apiserver/pkg/admission/plugin/webhook/mutating/plugin.go

```diff
@@ -69,7 +69,6 @@ func NewMutatingWebhook(configFile io.Reader) (*Plugin, error) {
 
 // SetScheme sets a serializer(NegotiatedSerializer) which is derived from the scheme
 func (a *Plugin) SetScheme(scheme *runtime.Scheme) {
-	a.Webhook.SetScheme(scheme)
 	if scheme != nil {
 		a.scheme = scheme
 		a.jsonSerializer = json.NewSerializer(json.DefaultMetaFactory, scheme, scheme, false)
```

Comment:
and I think we'd need a per-request serializer, which means we need the request scoped ObjectTyper